src/renderer/components/ProjectCreation/CloudConfigView.tsx

```diff
@@ -0,0 +1,168 @@
+import {
+  Box,
+  IconButton,
+  Link,
+  Stack,
+  styled,
+  TextField,
+  Typography,
+} from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import colors from 'renderer/colors';
+import { ChangeEvent, useState } from 'react';
+import useKeypress from 'renderer/utils/hooks';
+import { PrimaryButton, SecondaryButton } from '../Blocks/Buttons';
+import ipc from '../../ipc';
+
+const { openExternalLink } = ipc;
+
+interface Props {
+  prevView: () => void;
+  closeModal: () => void;
+  nextView: () => void;
+  projectName: string;
+  textToDisplay: string | null;
+}
+
+const CustomStack = styled(Stack)({ width: '100%' });
+
+const CustomColumnStack = styled(CustomStack)({ flexDirection: 'column' });
+
+const CustomRowStack = styled(CustomStack)({
+  flexDirection: 'row',
+  alignItems: 'center',
+});
+
+const Container = styled(Box)({
+  backgroundColor: colors.grey[700],
+});
+
+const CloudConfigView = ({
+  prevView,
+  closeModal,
+  nextView,
+  projectName,
+  textToDisplay,
+}: Props) => {
+  const [isAwaitingAPIKey, setAwaitingAPIKey] = useState<boolean>(true);
+  const [isAwaitingClientSecret, setAwaitingClientSecret] =
+    useState<boolean>(true);
+  const [APIKey, setAPIKey] = useState<string>('');
+  const [clientSecret, setClientSecret] = useState<string>('');
+
+  const saveCloudCredentials: () => void = () => {
+    setAPIKey(APIKey.trim());
+    setClientSecret(clientSecret.trim());
+    nextView();
+  };
+
+  const handleAPIKeyInput = (
+    event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    const key = event.target.value;
+    setAPIKey(key);
+    setAwaitingAPIKey(key.length === 0);
+  };
+
+  const handleClientSecretInput = (
+    event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    const secret = event.target.value;
+    setClientSecret(secret);
+    setAwaitingClientSecret(secret.length === 0);
+  };
+
+  const saveButton = (
+    <PrimaryButton
+      onClick={saveCloudCredentials}
+      disabled={isAwaitingAPIKey || isAwaitingClientSecret}
+      fullWidth
+    >
+      Save
+    </PrimaryButton>
+  );
+
+  const cancelButton = (
+    <SecondaryButton onClick={prevView} fullWidth>
+      Back
+    </SecondaryButton>
+  );
+
+  const handleHelpClick: () => void = () => {
+    openExternalLink('https://mlvet.app');
```

Comment:
Can we make this be declared somewhere else so we have all marketing / landing page related stuff in one place in case we change it (e.g. help page might go to mlvet.app/help in future)

Reply:
For now, im going to go with Rileys suggestion and just point [this tutorial](http://nerdvittles.com/creating-an-api-key-for-google-speech-recognition/).

Reply:
Done here: [0fbdc31](https://github.com/chloebrett/mlvet/pull/214/commits/0fbdc31db562590379be0dcb71febd9a5527f979)